src/libnnstreamer-edge/nnstreamer-edge-internal.c

```diff
@@ -1463,6 +1610,9 @@ nns_edge_release_handle (nns_edge_h edge_h)
     return NNS_EDGE_ERROR_INVALID_PARAMETER;
   }
 
+  if (eh->is_started)
```

Comment:
calling nns_edge_stop (eh) without eh->is_started (to avoid svace issue - set flag without lock)
check eh->is_started in nns_edge_stop()